documentation/guides-eslint-sonarqube-config.asciidoc

```diff
@@ -0,0 +1,41 @@
+:toc: macro
+
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+toc::[]
+:idprefix:
+:idseperator: -
+:reproducible:
+:source-highligter: rouge
+:listing-caption: Listing
+
+= Importing your ESLint reports into SonarQube
+
+With devon4node, you can easily generate ESLint reports and import them into SonarQube. All you need are the CLIs of the link:https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/[SonarQube Scanner] and link:https://eslint.org/docs/user-guide/command-line-interface[ESLint], which can be installed via npm:
+
+- `npm i eslint`
```

Comment:
You should never install eslint or sonar-scanner as regular dependency. Those packages are only required for development, not for production. So my recommendation is to install it as devDependency of global package. Another point, all devon4node projects has eslint as devDependency.